src/core/cm/launcher/balancer.hpp

```diff
@@ -84,19 +89,20 @@ class Balancer {
 
     Error PerformPolicyBalancing(const Array<RunInstanceRequest>& instances);
 
-    InstanceManager*                   mInstanceManager   = nullptr;
-    ImageInfoProviderItf*              mImageInfoProvider = nullptr;
-    NodeManager*                       mNodeManager       = nullptr;
-    MonitoringProviderItf*             mMonitorProvider   = nullptr;
-    networkmanager::NetworkManagerItf* mNetworkManager    = nullptr;
-    InstanceRunnerItf*                 mRunner            = nullptr;
+    ImageInfoProvider                  mImageInfoProvider;
+    InstanceManager*                   mInstanceManager = nullptr;
+    NodeManager*                       mNodeManager     = nullptr;
+    MonitoringProviderItf*             mMonitorProvider = nullptr;
+    networkmanager::NetworkManagerItf* mNetworkManager  = nullptr;
+    InstanceRunnerItf*                 mRunner          = nullptr;
```

Comment:
Update initialization for members
```suggestion
    InstanceRunnerItf*                 mRunner          {};
```